cfn-resources/trigger/cmd/resource/resource.go

```diff
@@ -21,6 +21,8 @@ import (
 	"fmt"
 	"log"
 
+	"github.com/mongodb/mongodbatlas-cloudformation-resources/profile"
+
```

Comment:
could you remove the newline and run go import?

Reply:
updated